spec/features/female_allocation_journey_feature_spec.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "womens allocation journey" do
+  let(:test_strategy) { Flipflop::FeatureSet.current.test! }
+  let(:prison) { build(:womens_prison) }
+  let(:offenders) { build_list(:nomis_offender, 5, complexityLevel: 'high') }
+  let(:offender) { build(:nomis_offender) }
+  let(:nomis_offender_id) { offender.fetch(:offenderNo) }
+  let(:user) { build(:pom) }
+  let(:probation_pom) { build(:pom, :probation_officer) }
+  let(:probation_pom2) { build(:pom, :probation_officer) }
+  let(:inactive_prison_pom) { build(:pom, :prison_officer) }
+  let(:prison_pom) { build(:pom, :prison_officer) }
+  let(:message_text) { Faker::Lorem.sentence }
+  let(:tiers) { ['A', 'B', 'C', 'D', 'N/A'].cycle.take(offenders.size) }
+
+  before do
+    test_strategy.switch!(:womens_estate, true)
+
+    create(:pom_detail, :inactive, nomis_staff_id: inactive_prison_pom.staff_id)
+    create(:pom_detail, :part_time, nomis_staff_id: probation_pom.staff_id)
+    create(:pom_detail, nomis_staff_id: probation_pom2.staff_id)
+
+    stub_signin_spo user, [prison.code]
+    stub_poms(prison.code, [probation_pom, probation_pom2, prison_pom, inactive_prison_pom])
+    stub_offenders_for_prison(prison.code, offenders + [offender])
+
+    offenders.each_with_index do |o, index|
+      ci = create(:case_information, tier: tiers[index], nomis_offender_id: o.fetch(:offenderNo))
+      create(:allocation, prison: prison.code, nomis_offender_id: ci.nomis_offender_id, primary_pom_nomis_id: probation_pom.staff_id)
+    end
+
+    create(:case_information, nomis_offender_id: nomis_offender_id)
+    alloc = create(:allocation, prison: prison.code, nomis_offender_id: nomis_offender_id, primary_pom_nomis_id: probation_pom.staff_id)
+    alloc.deallocate_offender_after_release
+    alloc.update! primary_pom_nomis_id: prison_pom.staff_id
+    alloc.deallocate_offender_after_release
+
+    visit unallocated_prison_prisoners_path prison.code
+    click_link 'Allocate'
+    # visit new_prison_prisoner_allocation_path(prison.code, nomis_offender_id)
+  end
+
+  after do
+    test_strategy.switch!(:womens_estate, false)
+  end
+
+  scenario 'accepting recommendation' do
```

Comment:
It might be nice to cover the unhappy path in this spec:

- If you leave the override reasons blank, you get validation errors